tide_site_restriction.module

```diff
@@ -208,12 +208,20 @@ function tide_site_restriction_node_grants(AccountInterface $account, $op) {
  * Implements hook_node_grants_alter().
  */
 function tide_site_restriction_node_grants_alter(&$grants, AccountInterface $account, $op) {
-  foreach ($grants as $realm => $gid) {
-    // Other modules should respect to the tide_site_restriction module.
-    if ($realm != 'tide_site_restriction') {
-      unset($grants[$realm]);
+  /** @var \Drupal\tide_site_restriction\Helper $site_restriction_helper */
+  $site_restriction_helper = \Drupal::service('tide_site_restriction.helper');
+  // If the user could not bypass the restriction, it should respect the
```

Comment:
I reckon this would be clearer if we did the below. Thoughts?

```
if ($site_restriction_helper->canBypassRestriction($account) === TRUE) {
  return $grants;
}

foreach ($grants as $realm => $gid) {
```

Reply:
Thanks @anthony-malkoun , Yes, agreed.